scripts/post_install_script.sh

```diff
@@ -74,19 +207,32 @@ modules:
           FC: gfortran
           F90: gfortran
           F77: gfortran
+    #openmpi:
```

Comment:
remove

Reply:
With the upgrade to 2.8.0, we'll uncomment this block to enable openmpi to better integrate with SLURM PMI3.